frontend/packages/console-shared/src/components/catalog/CatalogController.tsx

```diff
@@ -127,16 +128,22 @@ const CatalogController: React.FC<CatalogControllerProps> = ({
   }, [items, queryParams]);
 
   const catalogTypes: CatalogType[] = React.useMemo(() => {
-    const types = catalogExtensions
+    const types: CatalogType[] = catalogExtensions
       .map((extension) => ({
```

Comment:
:tada: yeah types!

It's fine here, just for your info, you can alternative add the "expected return type" of the map as argument to map like this:

```suggestion
    const types = catalogExtensions
      .map<CatalogType>((extension) => ({
```

What's the difference? You can see this when changing label to label2 for example in the inner function.

When you say `types` is of type `CatalogType[]` you'll get an error on the assignment to `types`.

When you say `map` should return a type `CatalogType` you'll get an error on the arrow function that returns the wrong data.

Just FYI. It's fine to keep the existing code!